src/pages/docs/providers/gpu/gpu-golem-live.md

```diff
@@ -0,0 +1,288 @@
+
+- ASRock: DEL or F2
+- ASUS: DEL or F2
+- Acer: DEL or F2
+- Dell: F12 or F2
+- ECS: DEL
+- Gigabyte DEL or F2
+- HP: F10
+- Lenovo: F2 or Fn + F2 (laptops), F1 (desktops), Enter + F1 (ThinkPads)
+- MSI: DEL
+- Microsoft Surface: press and hold the Power and Volume Up buttons
+- Samsung: F2
+- Toshiba: F2
+
+In the BIOS settings (for the MSI motherboard as an example), set the boot order so that the SSD is in the first position (before the disk with the system). This will cause the `gpu-golem-live` provider to boot first (provided it is connected to the USB port).
+
+![Bios settings 2](/gpu/bios_2.png)
+
+## 4. GPU provider configuration
+
+In the next step you will learn how to configure the `gpu-golem-live` provider. Remember that the configuration process is done only once, at the beginning and you can make any modifications later (see [FAQ](#frequently-asked-questions)).
+
+1.  **Select OS** - click `Enter` to confirm the selection **GOLEM GPU Live**
+
+    ![Bios settings 3](/gpu/bios_3.png)
+
+2.  **Welcome to the wizard!** - Click `OK` to proceed.
+
+    ![Setup GPU Golem Live 1](/gpu/setup_1.png)
+
+3.  **Terms of Use Acceptance** - select `YES` to confirm the terms of use. You can read the whole thing at your convenience at: [https://docs.golem.network/docs/golem/terms](https://docs.golem.network/docs/golem/terms)
+
+    ![Setup GPU Golem Live 2](/gpu/setup_2.png)
+
+4.  **Persistent Storage Configuration** - select a storage partition. A list of available partitions will be displayed and you will be able to select one of them to use as persistent storage. The selected storage partition will be used to store data related to `gpu-golem-live`.
+
+    ![Setup GPU Golem Live 3](/gpu/setup_3.png)
+
+5.  **Password Setup** - in the next step a password will be generated, which should be saved securely. You will need the password to connect to `gpu-golem-live` via SSH in the future.
+
+    ![Setup GPU Golem Live 4](/gpu/setup_4.png)
+
+6.  **Network Configuration** - the next step will display the IP address assigned to the provider to `gpu-golem-live`. Write it down to use in the future for SSH connection.
+
+    ![Setup GPU Golem Live 5](/gpu/setup_5.png)
+
+7.  **GLM (Golem Network Token) Configuration** - in the next step, enter the address of your own wallet.
+
+    ![Setup GPU Golem Live 6](/gpu/setup_6.png)
+
+8.  **Set your prices** - in the following steps, set the hourly rate (in GLM) and the initial price (in GLM). These values are necessary to participate in the Golem Network and set prices for sharing resources.
+
+    _GLM per hour means how much GLM you will receive for a full hour of calculations._
+
+    ![Setup GPU Golem Live 7](/gpu/setup_7.png)
+
+    _GLM init price means how much GLM you will receive for taking the order. It is recommended to set this value to 0, due to billing for rental time._
```

Comment:
Init price? Why do we need a new term for start price which is already used on the network?

Reply:
will not init, as on the screen is init., will keep order for a moment, tbc with